test/helpers-unstable_mapSliceZone.test.ts

```diff
@@ -0,0 +1,188 @@
+import { TestContext, expect, it, vi } from "vitest";
+
+import { unstable_mapSliceZone } from "../src";
+
+const generateTestData = (ctx: TestContext) => {
+	const model1 = ctx.mock.model.sharedSlice({
+		variations: [
+			ctx.mock.model.sharedSliceVariation({
+				primaryFields: { foo: ctx.mock.model.keyText() },
+				itemsFields: { bar: ctx.mock.model.keyText() },
+			}),
+		],
+	});
+	const model2 = ctx.mock.model.sharedSlice({
+		variations: [
+			ctx.mock.model.sharedSliceVariation({
+				primaryFields: { foo: ctx.mock.model.keyText() },
+				itemsFields: { bar: ctx.mock.model.keyText() },
+			}),
+		],
+	});
+
+	const sliceZone = [
+		ctx.mock.value.sharedSlice({ model: model1 }),
+		ctx.mock.value.sharedSlice({ model: model2 }),
+	];
+
+	return {
+		sliceZone,
+		model1,
+		model2,
+	};
+};
+
+it("maps a Slice Zone", async (ctx) => {
+	const { sliceZone, model1, model2 } = generateTestData(ctx);
+
+	const actual = await unstable_mapSliceZone(sliceZone, {
+		[model1.id]: () => ({ foo: "bar" }),
+		[model2.id]: () => ({ baz: "qux" }),
+	});
+
+	expect(actual).toStrictEqual([
+		{ __mapped: true, id: undefined, slice_type: model1.id, foo: "bar" },
+		{ __mapped: true, id: undefined, slice_type: model2.id, baz: "qux" },
+	]);
+});
+
+it("supports mapping functions that return undefined", async (ctx) => {
+	const { sliceZone, model1, model2 } = generateTestData(ctx);
+
+	const actual = await unstable_mapSliceZone(sliceZone, {
+		[model1.id]: () => void 0,
```

Comment:
You're that kind of guy now, I see 🕶️

Reply:
`void 0` is just so much cooler than `undefined`

Reply:
What about `[][0]` then? 🙃